desktop/ui/src/main/java/org/datacleaner/windows/ResultWindow.java

```diff
@@ -563,7 +563,7 @@ public void rowProcessingSuccess(AnalysisJob job, final RowProcessingMetrics met
                 logger.info("rowProcessingSuccess: {}", job.getDatastore().getName());
                 _progressInformationPanel.updateProgressFinished(metrics.getTable());
                 _progressInformationPanel.addUserLog(
-                        "Processing of " + metrics.getTable().getName() + " finished. Generating results...");
+                        "Processing of " + metrics.getTable().getName() + " is completed. Generating results...");
```

Comment:
I'm not sure if "is completed" is really a change here, because I think it carries the same meaning as "finished". Maybe it should be something like "successful".